pkg/asset/machines/openstack/machines.go

```diff
@@ -81,7 +81,7 @@ func provider(clusterID, clusterName string, platform *openstack.Platform, mpool
 		},*/
 		Image:          osImage,
 		CloudName:      platform.Cloud,
-		CloudsSecret:   &corev1.SecretReference{Name: cloudsSecret},
+		CloudsSecret:   &corev1.SecretReference{Name: cloudsSecret, Namespace: "kube-system"},
```

Comment:
nit: What do you think about putting kube-system in a `const` just like `cloudSecret`?